src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/CreateVariantIngestFiles.java

```diff
@@ -75,12 +75,18 @@
             optional = true)
     public File sampleMap;
 
-    @Argument(fullName = "sample-id",
-            shortName = "SID",
+    @Argument(fullName = "gvs-sample-id",
+            shortName = "GVSID",
             doc = "Sample id",
             optional = true)
     public Long sampleIdParam;
 
+    @Argument(fullName = "sample-name",
+            shortName = "SN",
+            doc = "Sample id",
```

Comment:
Some docs here would be really helpful I think for the constellation of parameters around sample-name, gvs-sample-id, sample-name-mapping (maybe even the same comment for all of these parameters).